src/contrib/torch/pt_call_tvm/RuntimeModuleWrapper.cc

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#include <dlpack/dlpack.h>
+#include <dmlc/memory_io.h>
+#include <tvm/runtime/module.h>
+#include <tvm/runtime/registry.h>
+#include <tvm/target/codegen.h>
+#include <tvm/target/target.h>
+
+#include <cstdio>
+#include <map>
+#include <string>
+#include <vector>
+
+#include "../../../runtime/graph_executor/graph_executor_factory.h"
+#include "../base64.h"
+
+struct ThreadLocalStore {
+  tvm::runtime::Module mod;
+  static ThreadLocalStore* ThreadLocal() {
+    thread_local ThreadLocalStore tls;
+    return &tls;
+  }
+};
+
+namespace tvm {
+namespace contrib {
+
+struct RuntimeModule {
+  tvm::runtime::Module mod;
+
+  RuntimeModule(tvm::runtime::Module mod) : mod(mod) {}
+};
+
+RuntimeModule* get_runtime_module() {
+  return new RuntimeModule(ThreadLocalStore::ThreadLocal()->mod);
+}
+
+void operator_module_forward(RuntimeModule* runtime_module, DLManagedTensor** inputs,
+                             size_t input_size) {
+  tvm::runtime::PackedFunc run = runtime_module->mod.GetFunction("__tvm_main__");
+
+  std::vector<TVMValue> tvm_values(input_size);
+  std::vector<int> tvm_type_codes(input_size);
+  tvm::runtime::TVMArgsSetter setter(tvm_values.data(), tvm_type_codes.data());
+  for (int k = 0; k < input_size; ++k) {
+    setter(k, &inputs[k]->dl_tensor);
```

Comment:
```suggestion
    setter(k, NDArray::FromDLPack(inputs[k]));
```
This is related to the comment that suggests removing manual call to the deleter. The idea is, it's better to let NDArray take the ownership of the DLManagedTensor and let it manage the lifetime of it. Passing a DLManagedTensor* to a function (e.g., an array of DLManagedTensor* is passed to`operator_module_forward`)  implies transferring the responsibility of destroying it to the callee.